tgui/packages/tgui/interfaces/NtosScipaper.js

```diff
@@ -0,0 +1,345 @@
+                author: value,
+              })}
+          />
+          <Button selected={et_alia === true} onClick={() => act('et_alia')}>
+            {'Multiple Authors'}
+          </Button>
+        </LabeledList.Item>
+        <LabeledList.Item label="Abstract">
+          <Input
+            fluid
+            value={abstract}
+            onChange={(e, value) =>
+              act('rewrite', {
+                abstract: value,
+              })}
+          />
+        </LabeledList.Item>
+        <LabeledList.Item label="Selected File">
+          <Dropdown
+            width="25rem"
+            options={Object.keys(file_list)}
+            displayText={selected_file ? selected_file : '-'}
+            onSelected={(ordfile_name) =>
+              act('select_file', {
+                selected_uid: file_list[ordfile_name],
+              })}
+          />
+        </LabeledList.Item>
+        <LabeledList.Item label="Selected Experiment">
+          <Dropdown
+            width="25rem"
+            options={Object.keys(exp_list)}
+            displayText={selected_experiment ? selected_experiment : '-'}
+            onSelected={(experiment_name) =>
+              act('select_experiment', {
+                selected_expath: exp_list[experiment_name],
+              })}
+          />
+        </LabeledList.Item>
+        <LabeledList.Item label="Selected Tier">
+          <Dropdown
+            width="25rem"
+            options={allowed_tiers.map((number) => String(number))}
+            displayText={tier ? String(tier) : '-'}
+            onSelected={(new_tier) =>
+              act('select_tier', {
+                selected_tier: Number(new_tier),
+              })}
+          />
+        </LabeledList.Item>
+        <LabeledList.Item label="Selected Partner">
+          <Dropdown
+            width="25rem"
+            options={Object.keys(allowed_partners)}
+            displayText={selected_partner ? selected_partner : '-'}
+            onSelected={(new_partner) =>
+              act('select_partner', {
+                selected_partner: allowed_partners[new_partner],
+              })}
+          />
+        </LabeledList.Item>
+      </LabeledList>
+    </Section>
+  );
+  const PaperResults = (
+    <Section title="Expected Results">
+      <Stack fill>
+        <Stack.Item grow>
+          {'Renown: '}
+          <BlockQuote>{gains[renown_index - 1]}</BlockQuote>
+        </Stack.Item>
+        <Stack.Item grow>
+          {'Cooperation: '}
+          <BlockQuote>{gains[coop_index - 1]}</BlockQuote>
+        </Stack.Item>
+        <Stack.Item grow>
+          {'Funding: '}
+          <BlockQuote>{gains[funding_index - 1]}</BlockQuote>
+        </Stack.Item>
+      </Stack>
+      <br />
+      <Button
+        icon="upload"
+        textAlign="center"
+        fluid
+        onClick={() => act('publish')}>
+        Publish Paper
+      </Button>
+    </Section>
+  );
+  return (
+    <>
+      {PaperForm}
```

Comment:
Should begin in lowercase, or better yet, just inline it.

Reply:
This has been largely unaddressed.

Reply:
Woops